genni.yml

```diff
@@ -1 +1 @@
-genni_home: "/home/isak/life/references/projects/phd/do-not-circulate/GENNI_public/"
```

Comment:
Optional: remove this commit from all history of the repo for privacy and add `genni.yml` to `.gitignore`

Reply:
Nothing private, I don't mind just keeping it.